src/main/kotlin/org/misarch/address/graphql/model/UserAddress.kt

```diff
@@ -0,0 +1,34 @@
+package org.misarch.address.graphql.model
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import com.expediagroup.graphql.generator.federation.directives.FieldSet
+import com.expediagroup.graphql.generator.federation.directives.KeyDirective
+import graphql.schema.DataFetchingEnvironment
+import org.misarch.address.graphql.dataloader.UserDataLoader
+import java.util.*
+import java.util.concurrent.CompletableFuture
+
+@GraphQLDescription("A address associated with a user.")
+@KeyDirective(fields = FieldSet("id"))
+class UserAddress(
+    id: UUID,
+    street1: String,
+    street2: String,
+    city: String,
+    postalCode: String,
+    country: String,
+    companyName: String,
+    internal val userId: UUID,
+    @property:GraphQLDescription("If true, this address is archived and can no longer be used.")
+    val isArchived: Boolean
+) : Address(id, street1, street2, city, postalCode, country, companyName) {
+
+    @GraphQLDescription("The user this notification was sent to.")
```

Comment:
notification :raised_eyebrow:?

Reply:
ok now I'm annoyed
I search for that shit with vs code

Reply:
fixed with 36e20e25747be298892815e3a7caed16db22cbe6